src/portal/src/i18n/lang/de-de-lang.json

```diff
@@ -881,6 +881,7 @@
         "VERIFY_REMOTE_CERT": "Zertifikat der Gegenstelle prüfen",
         "TOKEN_EXPIRATION": "Token-Ablauf (Minuten)",
         "AUTH_MODE": "Auth Mode",
+        "PRIMARY_AUTH_MODE": "Dieser Anmeldemodus wird zur Standardmethode für die Benutzeranmeldung. Der Anmeldebildschirm, in dem der Benutzer auswählt, ob er sich über den Identitätsanbieter oder über die lokale DB anmelden möchte, leitet den Benutzer automatisch an diesen Identitätsanbieter weiter. Die Anmeldung über die DB ist möglich, wenn die URL '/account/sign-in' explizit aufgerufen wird.",
```

Comment:
This is an error. Here shouldn't be the text for the tooltip. It should be the label name: **Primärer Anmeldemodus**

Reply:
L905 after auth_mode should contain the tooltip.


`        "PRIMARY_AUTH_MODE": "Dieser Anmeldemodus wird zur Standardmethode für die Benutzeranmeldung. Der Anmeldebildschirm, in dem der Benutzer auswählt, ob er sich über den Identitätsanbieter oder über die lokale DB anmelden möchte, leitet den Benutzer automatisch an diesen Identitätsanbieter weiter. Die Anmeldung über die DB ist möglich, wenn die URL '/account/sign-in' explizit aufgerufen wird."`